src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/AS_QualByDepth.java

```diff
@@ -0,0 +1,73 @@
+ * <p>QUAL/AD = 4107.13/356 = 11.54 = QD</p>
+ * <p>Note that currently, when HaplotypeCaller is run with `-ERC GVCF`, the QD calculation is invoked before AD itself has been calculated, due to a technical constraint. In that case, HaplotypeCaller uses the number of overlapping reads from the haplotype likelihood calculation in place of AD to calculate QD, which generally yields a very similar number. This does not cause any measurable problems, but can cause some confusion since the number may be slightly different than what you would expect to get if you did the calculation manually. For that reason, this behavior will be modified in an upcoming version.</p>
+ *
+ * <h3>Caveat</h3>
+ * <p>This annotation can only be calculated for sites for which at least one sample was genotyped as carrying a variant allele.</p>
+ *
+ * <h3>Related annotations</h3>
+ * <ul>
+ *     <li><b><a href="https://www.broadinstitute.org/gatk/guide/tooldocs/org_broadinstitute_gatk_tools_walkers_annotator_AS_QualByDepth.php">AS_QualByDepth</a></b> outputs a version of this annotation that includes all alternate alleles in a single calculation.</li>
+ *     <li><b><a href="https://www.broadinstitute.org/gatk/guide/tooldocs/org_broadinstitute_gatk_tools_walkers_annotator_Coverage.php">Coverage</a></b> gives the filtered depth of coverage for each sample and the unfiltered depth across all samples.</li>
+ *     <li><b><a href="https://www.broadinstitute.org/gatk/guide/tooldocs/org_broadinstitute_gatk_tools_walkers_annotator_DepthPerAlleleBySample.php">DepthPerAlleleBySample</a></b> calculates depth of coverage for each allele per sample (AD).</li>
+ * </ul>
+ */
+public class AS_QualByDepth extends InfoFieldAnnotation implements ReducibleAnnotation, AS_StandardAnnotation {
+
+    @Override
+    public List<String> getKeyNames() { return Arrays.asList(GATKVCFConstants.AS_QUAL_BY_DEPTH_KEY); }
+
+    @Override
+    public String getRawKeyName() { return GATKVCFConstants.AS_QUAL_KEY; }
+
+    @Override
+    public List<VCFInfoHeaderLine> getDescriptions() {
+        //We only have the finalized key name here because the raw key is internal to GenotypeGVCFs and won't get output in any VCF
+        return Arrays.asList(GATKVCFHeaderLines.getInfoLine(getKeyNames().get(0)));
+    }
+
+    @Override
+    public Map<String, Object> annotate(final ReferenceContext ref,
+                                        final VariantContext vc,
+                                        final Map<String, PerReadAlleleLikelihoodMap> perReadAlleleLikelihoodMap ) {
+        return null;
+    }
+
+    @Override
+    public Map<String, Object> annotateRawData(final ReferenceContext ref,
+                                               final VariantContext vc,
+                                               final Map<String, PerReadAlleleLikelihoodMap> stratifiedPerReadAlleleLikelihoodMap) {
+        return null;
+    }
+
+    @Override
+    @SuppressWarnings({"unchecked", "rawtypes"})//FIXME
```

Comment:
`FIXME` with no explanation -- briefly describe why you were unable to avoid using a raw type for posterity